frontend/src/components/ShowMoreButton.tsx

```diff
@@ -0,0 +1,40 @@
+import { faChevronDown, faChevronUp } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { Button } from '@heroui/button'
+import { useState } from "react"
+
+const ShowMoreButton = ({
+  onToggle,
+}: {
+  onToggle: () => void
+}) => {
+  const [isExpanded, setIsExpanded] = useState(false)
+
+  const handleToggle = () => {
+    setIsExpanded(!isExpanded)
+    onToggle()
+  }
+
+  return (
+    <div className="mt-4 flex justify-start">
+      <Button
+        type="button"
+        disableAnimation
+        onPress={handleToggle}
+        className="flex items-center bg-transparent text-blue-400"
```

Comment:
I thought the original issue was about misalighment.. but the only change is that you put the code for the button in a separate component (good for resuing it). But the alignment is still off, since those classes were not changed. 
<img width="584" height="246" alt="Screenshot 2025-07-30 at 9 34 54 PM" src="https://github.com/user-attachments/assets/82aea4bf-1043-46a9-8e23-a78c486beb43" />
<img width="1251" height="391" alt="Screenshot 2025-07-30 at 9 34 38 PM" src="https://github.com/user-attachments/assets/bdee66d4-1f95-4a5d-a462-260fb894eed7" />


The fix is actually a simple one line update 😅 
```suggestion
        className="pl-0 flex items-center bg-transparent text-blue-400"
```

Reply:
That’s a neat and efficient fix 😄 Appreciate the suggestion — makes the UI cleaner with just a small change. Thanks, @Kasya! 🙌